### PR TITLE
Fix small bugs of the language translation #164

### PR DESCRIPTION
**Pull request by TammamWafai**

### CODE REVIEW

File: src/components/shared/SideBar.tsx [hunk 1]
```diff
@@ -212,11 +211,12 @@ const SideBar = ({
...
-                  We didn't find any results
+                  {t("We didn't find any results")}
                 </p>
                 <p className="w-full text-center flex justify-center animate__animated animate__headShake">
-                  Make sure everything is spelled correctly or try different
-                  keywords
+                  {t(
+                    "Make sure everything is spelled correctly or try different keywords"
+                  )}
                 </p>
```
Comment (Said-laasri) [inline src/components/shared/SideBar.tsx:220]: great job on adding these variable

File: src/components/shared/NavBar.tsx [hunk 2]
```diff
@@ -108,66 +107,85 @@ const NavBar: FC<NavBarProps> = ({}) => {
...
-            {/* sign up button */}
+
+          {/* sign up button */}
           <Button
             type="button"
             className={`max-md:px-4 max-md:py-2 md:mr-4 px-7 py-2 rounded-[3px] text-black border border-[#000] ${
-              isDarkMode ? "border border-[#F5F5F5] text-white" : "border border-[#000] text-black"
+              isDarkMode
+                ? "border border-[#F5F5F5] text-white"
+                : "border border-[#000] text-black"
```
Comment (Said-laasri) [inline src/components/shared/NavBar.tsx:169]: for this icon dark mode and translation button, I removed the border and hover effect and the bg you can check on deployed version so no need anymore

File: src/components/shared/NavBar.tsx [hunk 3]
```diff
@@ -108,66 +107,85 @@ const NavBar: FC<NavBarProps> = ({}) => {
...
+              >
+                {/* map languages */}
+                {Object.entries(languages).map(
+                  ([languageName, languageCode]) => (
+                    <button
+                      key={languageCode}
+                      onClick={() => handleLanguageChange(languageCode)}
+                      className={` px-5 py-2 w-full 
+                      ${
+                        selectedLanguage === languageCode
```
Comment (Said-laasri) [inline src/components/shared/NavBar.tsx:149]: did you change the style of the language list ??

File: src/components/shared/LeftSideBar.tsx [hunk 4]
```diff
@@ -159,25 +172,39 @@ const LeftSideBar = ({
...
                 ${isDarkMode ? "text-white" : "text-[#25282C]"}`}
           />
           {showLanguages && (
-            <div ref={languageRef} className="overflow-hidden absolute z-50 bottom-[80px] bg-white border-[1px] border-gray-200 rounded-lg shadow-md flex flex-col items-center">
+            <div
+              ref={languageRef}
+              className={`overflow-hidden absolute z-50 bottom-[30px] ${
+                selectedLanguage === "ar" ? "right-[60px]" : "left-[60px]"
+              } bg-white border-[1px] border-gray-200 rounded-lg shadow-md flex flex-col items-center`}
+            >
```
Comment (Said-laasri) [inline src/components/shared/LeftSideBar.tsx:197]: did you change where the list language appears in the left sidebar just make sure it adapts to both Arabic and English so  opposite sides depending on language choice

**Review by Said-laasri — COMMENTED** [review]
Great Work @TammamWafai  !

i left some comments in code and here too please check both any question let me know on whatsapp thank you

 
*****************************************************************************
Please Update the sidebar text to adjust to dark mode because in dark mode it still in text-balck wich doesn't look visible 
**************************************************************************************

File: src/components/shared/NavBar.tsx (hunk 2 above)
Comment (TammamWafai) [reply src/components/shared/NavBar.tsx:169]: which branch?

File: src/components/shared/NavBar.tsx (hunk 3 above)
Comment (TammamWafai) [reply src/components/shared/NavBar.tsx:149]: Yes, I did as Tam had a screenshot of the selected language and how the font is white and the bg is grey. I fixed it in this PR

File: src/components/shared/LeftSideBar.tsx (hunk 4 above)
Comment (TammamWafai) [reply src/components/shared/LeftSideBar.tsx:197]: @Said-laasri did you test this PR? 
All the points you mentioned in this review are fixed including the positions of the languages list in both Arabic and other languages.

**Comment by TammamWafai** [comment]
Let me know if you have any other questions @Said-laasri

**Pull request closed without merging**